blog/retention analysis.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Cohort Analysis on Finance Application
+authors: [xiao fei]
```

Comment:
you need to add the author information in the "blog/authors.yml" file.
or it contains an error when executing "yarn start".